### DISCUSSION TITLE
Περιπτωση λαθασμενης  απαντησης στα video quiz

### DISCUSSION THREAD

**Discussion by giannisbarlas1** [courses-ionio/hci#1715] [Announcements]
Υπαρχει μια απορια σχετικα με το αν μπορει να διορθωση σε απαντηση στα video quiz και πιο συγκεκριμενα σε βιντεο που ειναι 
completed

**Comment by AlexandrosV20** [comment]
Οχι αν κάνεις submit μια ερώτηση δεν γίνεται να την αλλάξεις.

**Comment by p14kairas** [comment]
Εάν έχεις ολοκληρώσει το quiz και σου λεει "to be graded" τότε δεν μπορείς να αλλάξεις τις απαντήσεις σου.
